app/routines/update_user_contact_info.rb

```diff
@@ -121,6 +121,8 @@ def call
       user.grant_tutor_access = sf_contact.grant_tutor_access
 
       if school.nil? && !sf_school.nil?
+        # Add the possible updated school id to the record, we check for this in UpdateSchoolSalesforceInfo
+        user.school.updated_salesforce_id = sf_school.id
```

Comment:
user.school is maybe not always set?

Reply:
Also this code would only be relevant when the school is in SF but not in Accounts, which hopefully only happens sometimes in the first sync after CS adds a school in SF.